test/unit-test.el

```diff
@@ -1,5 +1,6 @@
 (require 'ert)
-(require 'groovy-mode)
+
+(load-file "groovy-mode.el")
```

Comment:
It is not clear to me that require will definitely pick up the correct groovy-mode.el. Also this change was made by someone else in a different Emacs mode project for this reason.